packages/compass-connections/src/stores/connections-store-redux.ts

```diff
@@ -1639,6 +1642,17 @@ const connectWithOptions = (
           );
         }
 
+        started = true;
+        log.info(
+          mongoLogId(1_001_000_005),
+          'Compass Connection Attempt Started',
+          'Connection attempt started',
+          {
+            clusterName: connectionInfo.atlasMetadata?.clusterName,
+            connectionId: connectionInfo.id,
+          }
+        );
```

Comment:
@syn-zhu  On line 1583, there is log `1_001_000_004` that carries the same info, is there a significance to this logs position over that one?